scripts/export-activity-to-rogue.php

```diff
@@ -0,0 +1,177 @@
+                      ORDER BY signups.sid')->fetchAll();
+}
+
+$client = dosomething_rogue_client();
+
+foreach ($signups as $signup) {
+  $data = [];
+  echo 'Trying sid ' . $signup->sid . '...' . PHP_EOL;
+
+  $northstar_id = dosomething_northstar_get_user($signup->uid, 'drupal_id');
+
+  // Only try to send to Rogue if we have a Northstar ID
+  if (isset($northstar_id)) {
+    // Match Rogue's timestamp format
+    $created_at = date('Y-m-d H:i:s', $signup->timestamp);
+
+    // Format signup data
+    $data = [
+      'northstar_id' => $northstar_id->id,
+      'campaign_id' => $signup->nid,
+      'campaign_run_id' => $signup->run_nid,
+      'do_not_forward' => TRUE,
+      'source' => $signup->source,
+      'created_at' => $created_at,
+      'updated_at' => $created_at, // Set updated same as created, will be overwritten if there was a RB submitted
+    ];
+
+    $fids = [];
+
+    // Include all the Reportback data and files if they exist
+    if ($signup->rbid) {
+      // Pull the quantity and why
+      $data['quantity'] = $signup->quantity;
+      $data['why_participated'] = $signup->why_participated;
+
+      // Match Rogue's timestamp format
+      $updated_at = date('Y-m-d H:i:s', $signup->updated);
+      $data['updated_at'] = $updated_at;
+
+      // Get the files for the Reportback
+      $photos = db_query('SELECT rbf.fid, rbf.remote_addr, rbf.caption, rbf.status, rbf.reviewed, rbf.reviewer, rbf.source, rbf.status, rblog.timestamp
+                          FROM dosomething_reportback_file rbf
+                          INNER JOIN dosomething_reportback_log rblog ON rbf.fid = substring_index(rblog.files, \',\',-1)
+                          WHERE rbf.rbid = ' . $signup->rbid . '
+                          GROUP BY rbf.fid')->fetchAll();
+
+      // Format the photos to send to Rogue
+      foreach ($photos as $key=>$photo) {
+        echo "\t" . 'Trying fid ' . $photo->fid . '...' . PHP_EOL;
+        array_push($fids, $photo->fid);
+
+        // If any of the Reportback files are not approved, send the quantity to Rogue as quantity_pending
+        $rogue_status = dosomething_rogue_transform_status($photo->status);
+        if ($rogue_status !== 'accepted') {
+          $data['quantity_pending'] = $signup->quantity;
+          $data['quantity'] = NULL;
+        }
+
+        // Match Rogue's timestamp format
+        $photo_created_at = date('Y-m-d H:i:s', $photo->timestamp);
+
+        // Format the photo data
+        $data['photo'][$key] = [
+          'source' => $photo->source,
+          'remote_addr' => $photo->remote_addr,
+          'caption' => $photo->caption,
+          'event_type' => 'post_photo',
+          'northstar_id' => $northstar_id->id,
+          'status' => $rogue_status,
+          'do_not_forward' => TRUE,
+          'submission_type' => 'user',
+          'file' => dosomething_helpers_get_data_uri_from_fid($photo->fid),
```

Comment:
@sbsmith86 hmm good question. I think it's globally available because this line was behaving as expected